CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
 
 ### Internal
 * Using Sync 10.1.0 and Core 10.1.0.
+* Analytics is now also upload to S3 service
```

Comment:
That sentence sounds a bit awkward - perhaps "Submit Analytics to S3/Segment in addition to Mixpanel".